src/components/layout/leadinfo-script.tsx

```diff
@@ -1,20 +1,30 @@
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
+
+let leadinfoInitialized = false;
+
+const LeadinfoScript = ({ enable }: { enable: boolean }) => {
+  const [script, setScript] = useState<HTMLScriptElement | null>(null);
```

Comment:
as said I think it's easier to use the script manager instead of manually bootstrapping it